### ISSUE TITLE
Average pooling

### ISSUE THREAD

**Issue by a462428** [huanghoujing/beyond-part-models#15]
Hi,

in paper, there is an average pooling before 1x1Conv

however, I don't know where is it in your model?  

self.base = resnet50(
pretrained=True,
last_conv_stride=last_conv_stride,
last_conv_dilation=last_conv_dilation)
self.num_stripes = num_stripes

self.local_conv_list = nn.ModuleList()
for _ in range(num_stripes):
     self.local_conv_list.append(nn.Sequential(
     nn.Conv2d(2048, local_conv_out_channels, 1),
     nn.BatchNorm2d(local_conv_out_channels),
     nn.ReLU(inplace=True)
))

**Comment by huanghoujing** [comment]
Thanks for your attention! The average pooling is [here](https://github.com/huanghoujing/beyond-part-models/blob/23d36d2e13be497feb9c28355a522a714a312460/bpm/model/PCBModel.py#L56).
